### ISSUE TITLE
how can i add headers in Web socket channel in flutter web 

### ISSUE THREAD

**Issue by princesoni18** [dart-lang/http#1695]
in mobile app we can use IoWebSocketChannel but its not supported in Flutter web ,

but I am unable to add Headers in WebsocketChannel ,  can anyone provide a solution

like we use in IO

        return IOWebSocketChannel.connect(
          url,
          protocols: protocols,
          headers: {'Authorization': "Bearer $accessToken"},
        );

**Comment by brianquinlan** [comment]
I don't think that it is possible to control the connection headers using the browser `WebSocket` API.

Someone please correct me (and reopen the issue) if I'm wrong.
